controller/local/controller.go

```diff
@@ -36,38 +40,63 @@ func (b *localController) Build(ctx context.Context, options controllerapi.Build
 	return b.ref, resp, nil
 }
 
-func (b *localController) Invoke(ctx context.Context, ref string, cfg controllerapi.ContainerConfig, ioIn io.ReadCloser, ioOut io.WriteCloser, ioErr io.WriteCloser) error {
+func (b *localController) ListProcesses(ctx context.Context, ref string) (infos []*controllerapi.ProcessInfo, retErr error) {
 	if ref != b.ref {
-		return errors.Errorf("unknown ref %q", ref)
+		return nil, errors.Errorf("unknown ref %q", ref)
```

Comment:
I assume that the local controller also doesn't support multiple build requests at a time? We should add an explicit error in this case as well, similar to https://github.com/docker/buildx/pull/1626#discussion_r1112960386.

Reply:
Fixed to return error on parallel builds.